graf3d/eve7/src/REveTypes.cxx

```diff
@@ -45,7 +46,7 @@ REveException REX::operator+(const REveException &s1, ElementId_t x)
 
 REX::RLogChannel &REX::REveLog()
 {
-   static RLogChannel sLog("ROOT.Eve");
+   thread_local RLogChannel sLog("ROOT.Eve");
```

Comment:
Are you sure about `thread_local`? 
Idea of log channel that you can configure it from any place - not only from current thread.

Reply:
Thank you Sergey! 
I have undone this change.
I have reviewed the logging part again. The thread local is the string that buffer messages for client info:
https://github.com/root-project/root/blob/master/graf3d/eve7/inc/ROOT/REveManager.hxx#L110
 https://github.com/root-project/root/blob/master/graf3d/eve7/src/REveManager.cxx#L1137.